dhis-2/dhis-services/dhis-service-field-filtering/pom.xml

```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
Apparently this module did not exist so I don't think we should create it or what is the idea here?

Reply:
oh, well if it did not exist I will remove it 👌🏻 good catch, thank you!

Reply:
At least this is what I think when the full pom is new. Can't think of why else it would be all new.